docs/troubleshooting/semgrep.md

```diff
@@ -27,13 +27,13 @@ To collect these logs, you need to upload them as an artifact. Modify your workf
 semgrep:
     name: semgrep with managed policy
     runs-on: ubuntu-20.04
+    container:
+      image: returntocorp/semgrep:0.85
     steps:
-      - uses: actions/checkout@v2
-        with:
-          submodules: recursive
-      - uses: returntocorp/semgrep-action@v1
-        with:
-          publishToken: ${{ secrets.SEMGREP_APP_TOKEN }}
+      - uses: actions/checkout@v3
+      - run: semgrep ci
+        env:
+          SEMGREP_APP_TOKEN: ${{ secrets.SEMGREP_APP_TOKEN }}
```

Comment:
@brendongo @underyx I hope this command is the one you requested to update in this document: https://docs.google.com/document/d/1pFL_JrJsc3Zc4CQpN7GZ6nEm15tTXQ25Fg5DI2ysf1c/edit